tests/ip_link.py

```diff
@@ -17,14 +17,15 @@ def runTest(self):
         board.expect(prompt)
         if "ip: not found" not in board.before:
             up_interfaces = re.findall('\d: ([A-Za-z0-9-\.]+)[:@].*state UP ', board.before)
+            ether_interfaces = re.findall('\d: ([A-Za-z0-9-\.]+)[:@].*\n.*link/ether ', board.before)
+            assert up_interfaces == ether_interfaces
```

Comment:
You're correct, I actually misread the assertion and thought I saw it doing assert len(up_interfaces) == len(ether_interfaces).